arch/platform/native/platform.c

```diff
@@ -64,6 +64,15 @@
 #include "net/ipv6/uip-ds6.h"
 #endif /* NETSTACK_CONF_WITH_IPV6 */
 
+#include "lib/assert.h"
+#include "lib/csprng.h"
+#ifdef __linux__
+#include <sys/random.h>
+#elif defined(__MACH__)
```

Comment:
There is a single mention of `__MACH__` in the tree right now, other places use `__APPLE__` to detect compilation for macOS. Can this place use `__APPLE__` too, or is that different from `__MACH__`?